src/lib/Scenes/Artwork/Components/ArtworkTombstone.tsx

```diff
@@ -0,0 +1,157 @@
+
+export interface ArtworkTombstoneProps {
+  artwork: ArtworkTombstone_artwork
+}
+
+export interface ArtworkTombstoneState {
+  showingMoreArtists: boolean
+}
+
+export class ArtworkTombstone extends React.Component<ArtworkTombstoneProps, ArtworkTombstoneState> {
+  state = {
+    showingMoreArtists: false,
+  }
+  handleTap(href: string) {
+    SwitchBoard.presentNavigationViewController(this, href)
+  }
+
+  showMoreArtists = () => {
+    this.setState({
+      showingMoreArtists: !this.state.showingMoreArtists,
+    })
+  }
+
+  renderSingleArtist(artist: Artist) {
+    return (
+      <React.Fragment>
+        <Flex flexDirection="row">
+          {this.renderArtistName(artist.name, artist.href)}
+          <Sans color={color("black60")} size="6" mx={1}>
+            &middot;
+          </Sans>
+          <Sans color={color("black60")} size="4">
+            Follow
+          </Sans>
+        </Flex>
+      </React.Fragment>
+    )
+  }
+
+  renderArtistName(artistName: string, href: string) {
+    return href ? (
+      <>
+        <TouchableWithoutFeedback onPress={this.handleTap.bind(this, href)}>
+          <Serif size="5t" weight="semibold">
+            {artistName}
+          </Serif>
+        </TouchableWithoutFeedback>
+      </>
+    ) : (
+      <>
+        <Serif size="5t" weight="semibold">
+          {artistName}
+        </Serif>
+      </>
+    )
+  }
+
+  renderMultipleArtists = () => {
+    const {
+      artwork: { artists },
+    } = this.props
+
+    const truncatedArtists = !this.state.showingMoreArtists ? artists.slice(0, 3) : artists
+    const artistNames = truncatedArtists.map((artist, index) => {
+      const artistNameWithComma = index !== artists.length - 1 ? artist.name + ", " : artist.name
+      return (
+        <React.Fragment key={artist.href}>{this.renderArtistName(artistNameWithComma, artist.href)}</React.Fragment>
+      )
+    })
+
+    return (
+      <Flex flexDirection="row" flexWrap="wrap">
+        {artistNames}
+        {!this.state.showingMoreArtists &&
+          artists.length > 3 && (
+            <TouchableWithoutFeedback onPress={this.showMoreArtists}>
+              <Serif size="5t" weight="semibold">
+                {artists.length - 3} more
+              </Serif>
+            </TouchableWithoutFeedback>
+          )}
+      </Flex>
+    )
+  }
+
+  renderCulturalMaker(culturalMaker: string) {
+    return (
+      <Serif size="5t" weight="semibold">
+        {culturalMaker}
+      </Serif>
+    )
+  }
+
+  render() {
+    const { artwork } = this.props
+    return (
+      <Box textAlign="left">
+        <Flex flexDirection="row" flexWrap="wrap">
+          {artwork.artists.length === 1 ? this.renderSingleArtist(artwork.artists[0]) : this.renderMultipleArtists()}
+          {artwork.artists.length === 0 && artwork.cultural_maker && this.renderCulturalMaker(artwork.cultural_maker)}
+        </Flex>
+        <Serif color={color("black60")} size="3t" m="0" p="0">
```

Comment:
The line spacing / font sizes here are still a bit off. I made a quick mockup to compare the spec with the current implementation, using the x-height of the artist name to calibrate the scaling:

![image](https://user-images.githubusercontent.com/1242537/58250067-71b82b80-7d58-11e9-87d2-a1f2830ac240.png)

And then again using the x-height of the detail text to calibrate:

![image](https://user-images.githubusercontent.com/1242537/58250315-09b61500-7d59-11e9-999a-2f68c973592a.png)

So we can see that the font sizes are too far apart, and the line spacing is still slightly too large on the detail text, although that might be an issue with palette 😬 so maybe we can add another ticket to our backlog to investigate palette text sizes on mobile web vs react native to make things consistent @damassi do you know whether that work has been thought about/attempted before?

Reply:
@ds300 good find, and let's definitely investigate in a separate ticket as it seems outside the scope of this PR.

Reply:
I'm curious how this shook out on the local discovery side during their build out /cc @l2succes.